src/workerd/api/tail-worker-test.js

```diff
@@ -55,8 +55,12 @@ export default {
         event.event.type == 'attributes' ||
         event.event.type == 'spanClose'
       ) {
-        // Every SpanClose event must point to the spanId of a previously opened span.
-        assert.ok(spanIdSet.has(event.spanContext.spanId));
+        // Every SpanClose/attributes event must point to the spanId of a previously opened span
```

Comment:
Yup, that's a good description of the new behavior, thank you for updating the test and comment.